lib/auth.js

```diff
@@ -1,4 +1,4 @@
-function getSocketSignature(token, channel, socketID, data) {
+function getSocketSignature(pusher, token, channel, socketID, data) {
```

Comment:
I think this should be renamed to something like `generateAuthenticationResponse`.

Reply:
Agree in general but I would rather not do that refactoring in this PR

Reply:
Fair enough, since the name was previously not accurate, but I do think the changes in this PR make the name even less accurate.